presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+import java.util.Optional;
+
+public final class ColumnStatistics
+{
+    public static final ColumnStatistics EMPTY_STATISTICS = ColumnStatistics.builder().build();
+
+    private final Optional<StatisticsValue<Long>> dataSize;
```

Comment:
Another alternative would be to have `reliability` as separate field. All stats values could be `double` with `NaN` when stats is missing. Direct `double` values are easier to work with. Additionally, `NaNs` propagate nicely so formulas are compact.